benchmark_runner/common/oc/oc.py

```diff
@@ -598,7 +598,7 @@ def wait_for_ready(self, label: str, run_type: str = 'pod', workload: str = '',
             if 'met' in result.decode("utf-8"):
                 return True
         except Exception as err:
-            if 'pod' in workload:
+            if 'vm' not in workload:
```

Comment:
Suggest doing this the way you did the previous change:

```
            if 'vm' in workload:
                raise VMNotReadyTimeout(workload=workload)
            else:
                raise PodNotReadyTimeout(workload=workload)
```

just in case we wind up with a workload whose name actually contains 'pod'.  Not a major issue.